src/sky/pro/lesson3/HW4.java

```diff
@@ -0,0 +1,105 @@
+package sky.pro.lesson3;
+
+public class HW4 {
+    public static void main(String[] args) {
+        task1();
+        task2();
+        task3();
+        task4();
+        task5();
+    }
+
+    public static void task1() {
+        //Задача 1
+        System.out.println("Задача 1");
+        int clientOS = 0;
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        else if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+    }
+
+    public static void task2() {
+        //Задача 2
+        System.out.println("Задача 2");
+        int clientOS = 1;
+        int clientDeviceYear = 2015;
+        if (clientOS == 1 && clientDeviceYear >= 2015) {
+            System.out.println("Утсановите актуальную версию Android приложения");
+        } else if (clientOS == 1 && clientDeviceYear < 2015) {
+            System.out.println("Установите упрощенную версию Android приложения");
+        }
+        if (clientOS == 0 && clientDeviceYear >= 2015) {
+            System.out.println("Установите актуальную версию iOS приложения");
+        } else if (clientOS == 0 && clientDeviceYear < 2015)
+            System.out.println("Установите упрощенную версию iOS приложения");
+        else
+            System.out.println("Данная версия ОС не поддерживается");
+    }
+
+    public static void task3() {
+        //Задача 3
+        System.out.println("Задача 3");
+        int year = 2021;
+        if (year % 400 == 0) {
+            System.out.println("Год високосный");
+        } else
+        if (year % 100 == 0) {
```

Comment:
```suggestion
        } else if (year % 100 == 0) {
```
данную конструкцию принято писать на одной строке